sys/callout.c

```diff
@@ -62,16 +62,17 @@ void callout_stop(callout_t *handle) {
  * position.
 */
 void callout_process(realtime_t time) {
-  int now = time % CALLOUT_BUCKETS;
-  bool done = false;
-
-  while (!done) {
-    int last = ci.last++ % CALLOUT_BUCKETS;
-
-    if (last == now)
-      done = true;
-
-    callout_head_t *head = &ci.heads[last];
+  unsigned int current_bucket = ci.last % CALLOUT_BUCKETS, last_bucket;
```

Comment:
Let's set up a rule then... multiple declarations on single line is ok, but only one definition, e.g.:

```C
unsigned int last_bucket, first_bucket;
unsigned int current_bucket = ci.last % CALLOUT_BUCKETS;
```